LunaDll/IPC/IPCPipeServer.cpp

```diff
@@ -139,6 +139,12 @@ void IPCPipeServer::ReadThread()
         std::string pktMethod = pkt["method"];
         json pktId = pkt["id"];
         json pktParams = pkt["params"];
+        
+        if(pktMethod == "sendItemPlacing")
+        {
+            std::string result = pkt["params"].dump();
+            editorPlacedItem = result;
+        }
```

Comment:
This is not a correct way of extending `IPCPipeServer::ReadThread()`. You should be able to just.... use the `mCallbacks` map and a function like every other method does. I see no reason for this method to be a special case out of the blue. It's just extra complication to add more different ways of doing something.